addons/mass_mailing_event/views/event_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">event.event</field>
         <field name="inherit_id" ref="event.view_event_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//button[@name='button_cancel']" position="after">
+            <xpath expr="//header" position="inside">
```

Comment:
Don't use "inside" as order is not guaranteed. I bet it could be put before or after something.

Reply:
@tde-banana-odoo I can't :/ because in event, there is nothing in the header, but `website_event` add a button "Preview badge", and `mass_mailing_event` add the button "Contact attendee", but they do not depends on each other :/